### PR TITLE
[Delivers #89626204] remove /send_cart endpoint

### PR DESCRIPTION
**Pull request by afeld**
@phirefly We're sure Navigator doesn't need it anymore? It's still referenced in their code:

https://github.com/GSA/CAP-ACQUISITION_NAVIGATOR/search?utf8=%E2%9C%93&q=send_cart

### CODE REVIEW

**Comment by phirefly** [comment]
Yes. Per my last convo with CAP folks, we can remove this. Most of their links are pointing to old servers. I will contact them to let them know the endpoints are no longer active.

File: doc/overview.md [hunk 1]
```diff
@@ -73,7 +73,7 @@ The NCR use case was built around GSA service centers (paint shops, landscapers,
 
 ### [Navigator](https://github.com/GSA/CAP-ACQUISITION_NAVIGATOR)
 
-Their application initiates requests through the `/send_cart` API. They use the parallel flow, and specify approval groups rather than individual approvers.
+This is an old use case, which is no longer active.
```
Comment (cmc333333) [inline doc/overview.md]: Maybe remove the header altogether?

**Comment by cmc333333** [comment]
This is pretty massive, but checks out for me. I'd like a second pair of eyes to skim over the code before merging, though. @phirefly ?

**Comment by phirefly** [comment]
:shipit:

**Pull request merged**